sdk/keyvault/keyvault-admin/package.json

```diff
@@ -0,0 +1,109 @@
+{
+  "name": "@azure/keyvault-admin",
```

Comment:
Would it make sense to use the full name `@azure/keyvault-administration` or is abbreviation intentionally done to be idiomatic?

Reply:
Oh the abbreviation comes from a survey I did with the TypeScript team. They prefer `keyvault-admin`. I think I can challenge that if you and @schaabs  agree that it should be `keyvault-administration`.